src/app/legacy/containers/StoryPromo/LinkContents/index.jsx

```diff
@@ -30,13 +30,14 @@ const LinkContents = ({ item, isInline, id }) => {
       return 'photogallery';
     }
 
-    const mediaType = pathOr(null, ['media', 'format'], item);
-
+    let mediaType =
+      pathOr(null, ['media', 'format'], item) ||
+      pathOr(null, ['contentType'], item);
+    mediaType = mediaType.toLowerCase();
```

Comment:
We can combine this into 1 statement
```suggestion
    const mediaType =
     (pathOr(null, ['media', 'format'], item) ||
      pathOr(null, ['contentType'], item))?.toLowerCase();
```